components/service-binding-usage-controller/README.md

```diff
@@ -1,8 +1,8 @@
-# Binding Usage Controller
```

Comment:
What about convention of component's name  in `resources/*/values.yaml` files. For example for this component version could be found in `resources/service-catalog-addons/values.yaml` under the key `binding_usage_controller`. Should the key change?

Reply:
discussed, we will change the key and chart name.